mod_ci/controllers.py

```diff
@@ -62,7 +62,8 @@ def before_app_request():
         ]
     )
     if 'config' in g.menu_entries and 'entries' in config_entries:
-        g.menu_entries['config']['entries'] = config_entries['entries'] + g.menu_entries['config']['entries']
+        g.menu_entries['config']['entries'] = config_entries['entries'] + \
```

Comment:
Why again all these changes? Please stop changing these **unneeded** things

Reply:
I don't know how I'm doing that. I swear to god I'm doing this by accident and not noticing them